app/src/main/java/com/example/parkingsystem/mvp/model/ParkingModel.java

```diff
@@ -15,4 +16,10 @@ public void setSpaces(int spaces) {
     public int getSpaces() {
         return spaces;
     }
+
+    @Override
+    public int releaseParking() {
+        ReservationInformationDB reservationInformationDB = ReservationInformationDB.getInstanceDB();
+        return reservationInformationDB.releasePastReservations();
```

Comment:
you can do this here 
`return ReservationInformationDB.getInstanceDB().releasePastResevations()`. 
The variable is not necessary you are using in only one part

Reply:
done